Beatmap/src/MapDatabase.cpp

```diff
@@ -417,6 +416,14 @@ class MapDatabase_Impl
 				m_database.Exec("UPDATE Scores SET window_slam=84");
 				gotVersion = 19;
 			}
+			if (gotVersion == 19)
```

Comment:
I don't really see the reason for updating the db and also altering past scores.

Reply:
I thought courses check the existing timing windows to see if the scores are eligible to be saved or not?

Reply:
Actually I think I see your point, nvm

Reply:
Removed.